contracts/Comptroller/Comptroller.sol

```diff
@@ -1178,40 +1207,69 @@ contract Comptroller is ComptrollerV9Storage, ComptrollerInterfaceG2, Comptrolle
     function _become(Unitroller unitroller) external {
         require(msg.sender == unitroller.admin(), "only unitroller admin can");
         require(unitroller._acceptImplementation() == 0, "not authorized");
+
+
+        // TODO: Remove this post upgrade
```

Comment:
Should we add this to the upgrade ticket?

Reply:
I have added a check for now to execute _upgradeSplitVenusRewards function for once. And yeah we should remove this function in future to reduce comptroller size.